rdrf/rdrf/templates/rdrf_cdes/banner/_patient.html

```diff
@@ -0,0 +1,11 @@
+{% if patient_link or patient %}
```

Comment:
What's the significance of the template name's leading underscore, and directory? Supporting more banner types in the future?

Reply:
It was an attempt to start some general organisation of the templates in the rdrf_cdes directory. The leading underscore indicates that the template is a 'partial' template (e.g. not a full page). I've used this in other frameworks, but I don't think it's particularly used by Django. 

Happy to change this though if you would prefer?

Reply:
Ok cool, sounds like a good way to lay things out :+1: